### PR TITLE
Update emulator to version 28.0.25

### PR DESCRIPTION
**Pull request by grendello**
This version supports running in headless mode which may be useful for our CI
servers when running unit tests.

### CODE REVIEW

File: Configuration.props [hunk 1]
```diff
@@ -135,7 +135,7 @@
     <AndroidToolPath Condition=" '$(AndroidToolPath)' == '' ">$(AndroidSdkFullPath)\tools</AndroidToolPath>
     <AndroidToolsBinPath Condition=" '$(AndroidToolsBinPath)' == '' ">$(AndroidToolPath)\bin</AndroidToolsBinPath>
     <AndroidToolExe Condition=" '$(AndroidToolExe)' == '' ">android</AndroidToolExe>
-    <EmulatorVersion Condition=" '$(EmulatorVersion)' == '' ">5264690</EmulatorVersion>
+    <EmulatorVersion Condition=" '$(EmulatorVersion)' == '' ">5395263</EmulatorVersion>
     <EmulatorToolPath Condition=" '$(EmulatorToolPath)' == '' ">$(AndroidSdkFullPath)\emulator</EmulatorToolPath>
```
Comment (jonpryor) [inline Configuration.props]: If we want to try the headless emulator, shouldn't we *also* modify this line to use `emulator-headless`?

File: Configuration.props [hunk 2]
```diff
@@ -135,7 +135,7 @@
...
     <EmulatorToolExe Condition=" '$(EmulatorToolExe)' == '' ">emulator</EmulatorToolExe>
```
Comment (jonpryor) [inline Configuration.props]: If we want to try the headless emulator, shouldn't we *also* modify this line to use `emulator-headless`?

Comment (grendello) [reply Configuration.props]: I didn't want to do it in this PR - it is to bring the new version in, then we should test it locally and only then make the changes here.

Comment (grendello) [reply Configuration.props]: And it seems we need to get #3000 in first, to get real test results here...

Comment (dellis1972) [reply Configuration.props]: Looks like the `5395263` version of the emulator-darwin-5395263.zip does NOT contain `emulator-headless` :/

**Pull request merged**